content/docs/examples/multicluster/icp/index.md

```diff
@@ -0,0 +1,219 @@
+    environment and may not overlap. This can be configured by `network_cidr` and `service_cluster_ip_range` in `cluster/config.yaml`.
+    
+    {{< text bash >}}
+    ## Network in IPv4 CIDR format
+    network_cidr: 10.1.0.0/16
+
+    ## Kubernetes Settings
+    service_cluster_ip_range: 10.0.0.1/24
+    {{< /text >}}
+
+1.  After IBM Cloud Private cluster install finsihed, validate `kubectl` access to each cluster. Suppose we have
+    two clusters: cluster-1 and cluster-2.
+
+    1.  Follow steps [here](https://www.ibm.com/support/knowledgecenter/SSBS6K_2.1.0.3/manage_cluster/cfc_cli.html)
+        to config `kubectl` for cluster-1, and run following command to check cluster status.
+
+        {{< text bash >}}
+        $ kubectl get nodes
+        $ kubectl get pods --all-namespaces
+        {{< /text >}}
+
+    1.  Use same steps as cluster-1 above to check cluster-2.
+    
+## Config Pod Communication Cross IBM Cloud Private Clusters
+
+IBM Cloud Private is using Calico Node-to-Node Mesh by default to manage container networks. The BGP Client
+on each node distributes the IP router information to all nodes.
+
+In order to make sure pods can communicate across different clusters, you need to configure IP routers on
+all nodes in the cluster. You need to add IP routers in cluster-2 to cluster-1, and add IP routers in
+cluster-1 to cluster-2.
+
+You can check how to add IP routers from cluster-1 to cluster-2. With Node-to-Node Mesh mode, each node will
+have IP routers connecting to peer nodes in the cluster. In this example, both cluster has three nodes.
+
+For cluster-1, the `hosts` file is as follows:
+
+{{< text bash >}}
+9.111.255.21 gyliu-icp-1
+9.111.255.129 gyliu-icp-2
+9.111.255.29 gyliu-icp-3
+{{< /text >}}
+
+For cluster-2, the `hosts` file is as follows:
+
+{{< text bash >}}
+9.111.255.152 gyliu-ubuntu-3
+9.111.255.155 gyliu-ubuntu-2
+9.111.255.77 gyliu-ubuntu-1
+{{< /text >}}
+
+1.  Get all of the `ip route` on nodes in cluster-1 with command `ip route | grep bird`. See the following
+    example. In this example, both cluster has three nodes.
+
+    {{< text bash >}}
+    root@gyliu-icp-1:~# ip route | grep bird
+    10.1.43.0/26 via 9.111.255.29 dev tunl0 proto bird onlink
+    10.1.158.192/26 via 9.111.255.129 dev tunl0 proto bird onlink
+    blackhole 10.1.198.128/26 proto bird
+    {{< /text >}}
+    {{< text bash >}}
+    root@gyliu-icp-2:~# ip route | grep bird
+    10.1.43.0/26 via 9.111.255.29 dev tunl0  proto bird onlink
+    blackhole 10.1.158.192/26  proto bird
+    {{< /text >}}
+    {{< text bash >}}
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    root@gyliu-icp-3:~# ip route | grep bird
+    blackhole 10.1.43.0/26  proto bird
+    10.1.158.192/26 via 9.111.255.129 dev tunl0  proto bird onlink
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    {{< /text >}}
+
+1.  You can see there are three IP routers total for those three nodes in cluster-1.
```

Comment:
Remove "you can see" and capitalze "There"

Reply:
Done